### PR TITLE
Makefile cleanup, quieter splat, assist.py

### PR DESCRIPTION
**Pull request by ethteck**

### CODE REVIEW

File: src/world/area_sbk/sbk_01.c [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #include "common.h"
 
+//#include "world/common/SpawnSunEffect.c"
```
Comment (pixel-stuck) [inline src/world/area_sbk/sbk_01.c:3]: ?

File: src/world/area_sbk/sbk_00/sbk_00_2.c [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#include "common.h"
+
+//INCLUDE_ASM(s32, "world/area_sbk/sbk_00", func_80240040_929270);
```
Comment (pixel-stuck) [inline src/world/area_sbk/sbk_00/sbk_00_2.c:3]: remove dis

**Review by pixel-stuck — CHANGES_REQUESTED** [review]

**Review by nanaian — CHANGES_REQUESTED** [review]
I believe we agreed on `.inc.c` as the extensions for these.

File: src/world/area_sbk/sbk_01.c (hunk 1 above)
Comment (ethteck) [reply src/world/area_sbk/sbk_01.c:3]: I can't add this yet due to files not being split yet. This'll be fixed today

**Review by nanaian — APPROVED** [review]

**Review by pixel-stuck — APPROVED** [review]

**Pull request merged**